server/src/test/java/io/pravega/schemaregistry/storage/impl/group/GroupPravegaTest.java

```diff
@@ -0,0 +1,757 @@
+        SchemaInfo schemaInfo = new SchemaInfo("anygroup", SerializationFormat.Custom, ByteBuffer.wrap(schemaData),
+                ImmutableMap.of());
+        Etag etag = pravegaKVGroups.getGroup(null, groupName).join().getCurrentEtag().join();
+        pravegaKVGroups.getGroup(null, groupName).join().addSchema(schemaInfo, groupProperties, etag).join();
+        GroupsValue gv = tableStore.getEntry(GROUPS, new NamespaceAndGroup(null, groupName).toBytes(),
+                GroupsValue::fromBytes).join().getRecord();
+        // one schema
+        //versionInfo key
+        TableRecords.SchemaRecord schemaRecord = tableStore.getEntry(
+                "_schemaregistry" + String.format("/%s.#.metadata/0", gv.getId()),
+                new TableKeySerializer().toBytes(new TableRecords.VersionKey(0)),
+                x -> TableRecords.fromBytes(TableRecords.VersionKey.class, x,
+                        TableRecords.SchemaRecord.class)).join().getRecord();
+        assertEquals("anygroup", schemaRecord.getSchemaInfo().getType());
+        //schemaInfo key
+        TableRecords.SchemaVersionList schemaVersionList = tableStore.getEntry(
+                "_schemaregistry" + String.format("/%s.#.metadata/0", gv.getId()), new TableKeySerializer().toBytes(
+                        new TableRecords.SchemaFingerprintKey(HASH.hashBytes(schemaInfo.getSchemaData()).asLong())),
+                x -> TableRecords.fromBytes(TableRecords.SchemaFingerprintKey.class, x,
+                        TableRecords.SchemaVersionList.class)).join().getRecord();
+        assertEquals(1, schemaVersionList.getVersions().size());
+        assertEquals("anygroup", schemaVersionList.getVersions().get(0).getType());
+        //LatestSchemaVersionKey
+        TableRecords.LatestSchemaVersionValue latestSchemaVersionValue = tableStore.getEntry(
+                "_schemaregistry" + String.format("/%s.#.metadata/0", gv.getId()),
+                new TableKeySerializer().toBytes(LATEST_SCHEMA_VERSION_KEY),
+                x -> TableRecords.fromBytes(TableRecords.LatestSchemaVersionKey.class, x,
+                        TableRecords.LatestSchemaVersionValue.class)).join().getRecord();
+        assertEquals("anygroup", latestSchemaVersionValue.getVersion().getType());
+        assertEquals(0, latestSchemaVersionValue.getVersion().getId());
+        //LatestSchemaVersionForTypeKey
+        TableRecords.LatestSchemaVersionValue latestSchemaVersionValue1 = tableStore.getEntry(
+                "_schemaregistry" + String.format("/%s.#.metadata/0", gv.getId()), new TableKeySerializer().toBytes(
+                        new TableRecords.LatestSchemaVersionForTypeKey(schemaInfo.getType())),
+                x -> TableRecords.fromBytes(TableRecords.LatestSchemaVersionForTypeKey.class, x,
+                        TableRecords.LatestSchemaVersionValue.class)).join().getRecord();
+        assertTrue(latestSchemaVersionValue.equals(latestSchemaVersionValue1));
```

Comment:
here and many other places, assertTrue on equality can be replaced by assertEquals.